src/main/frontend/util/marker.cljs

```diff
@@ -1,14 +1,17 @@
 (ns frontend.util.marker
   (:require [clojure.string :as string]
-            [frontend.util :as util]))
+            [frontend.util :as util]
+            [frontend.state :as state]))
 
 (def marker-pattern
-  #"^(NOW|LATER|TODO|DOING|DONE|WAITING|WAIT|CANCELED|CANCELLED|STARTED|IN-PROGRESS)?\s?")
+  (let [format (state/get-preferred-format)]
```

Comment:
Agree, should be the block's format property instead of the "preferred format".

Reply:
@logseq-cldwalker @cnrpman Thanks for you comments. I've made `marker-pattern` to a function and added `page-format` for format check to avoid time-tracking `# TODO aaa` not working when switching from `markdown` to `org`.